src/routes/route.py

```diff
@@ -1,67 +1,95 @@
+import logging
+
 import geopandas as gpd
+import numpy as np
+from numpy.typing import NDArray
 from tqdm import tqdm
 
 from data_loader.population.utils import NODE_ID, POPULATION
 from routes.route_utils import path
 
 
 class Route:
-    def __init__(self, route_path: path, num_people_on_route: int) -> None:
+    def __init__(
+        self, route_path: path, num_people_on_route: int, departure_times: list[int]
+    ) -> None:
+        if len(departure_times) != num_people_on_route:
+            logging.fatal(
+                "Number of departure times must equal the number of people on route."
+            )
         self.path = route_path
         self.num_people_on_route = num_people_on_route
-        self.departure_times: dict[int, int] = {0: num_people_on_route}
+        self.departure_times = departure_times
 
-    def introduce_departure_times(self, chunks: int, interval: int) -> None:
-        """
-        Introduces different departure times for the persons taking the given route.
 
-        :param chunks: In how many different chunks should the persons on the route depart.
-        :param interval: How long should the interval between each departure be. Given in seconds.
-        :return: A dictionary of departure times where keys are seconds after midnight and values are the number of people departing on that time.
-        """
-        departure_times: dict[int, int] = {}
-        people_per_interval = int(self.num_people_on_route / chunks)
-        remaining_people = self.num_people_on_route % chunks
+def _departure_times(total_population: int, start: int, end: int) -> NDArray[np.int_]:
+    """
+    Introduces different departure times for the persons taking the given route, spread on a normal distribution
+    between the given start and end times.
+    :param total_population: The total population.
+    :param start: start of normal distribution. Given in seconds.
+    :param end: end of normal distribution. Given in seconds.
+    """
+    mean = (start + end) / 2  # mean
```

Comment:
Hmm, yes, the floor here is made out of floor

```suggestion
    mean = (start + end) / 2
```